src/main/java/com/thebois/models/world/structures/AbstractStructure.java

```diff
@@ -95,4 +101,9 @@ public boolean isCompleted() {
         return Optional.empty();
     }
 
+    private void postSpawnPawnsEvent(final Position spawnPosition) {
+        final SpawnPawnsEvent spawnPawnsEvent = new SpawnPawnsEvent(2, spawnPosition, 2f);
+        Pawntastic.getEventBus().post(spawnPawnsEvent);
+    }
```

Comment:
This is what I was talking about earlier, that it's really weird for a structure to tell the colony how it should spawn its pawns, and where they should be. All the structure should do is notify the colony that it has been placed, and colony should take it from there.